third_party/wabt/include/wabt/interp/value.h

```diff
@@ -0,0 +1,120 @@
+/*
```

Comment:
Why this needs to be moved out to a separate file?

Reply:
`interp.h` includes `jit.h` to be able to use the `JITFunction` class defined there, and to be able to get the types of the values that need to be returned, `jit.h` needs to use `Value` and `ValueTypes`, which were in `interp.h`

but this causes a circular include where `jit.h` includes `interp.h` and `interp.h` includes `jit.h`

this is why I moved them to a separate file, which can be included both in `interp.h` and `jit.h`

Reply:
The `interp.h` is not created by us, so we should invent something here. What about a forward definition of `ValueTypes` and others?